src/AllCapsules.jsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+import getCapsules from "./helpers/api"
+import styles from "./AllCapsules.module.css"
+import rocketActive from "./assets/rocketActive.svg"
+import rocketUnknown from "./assets/rocketUnknown.svg"
+import rocketDeactivate from "./assets/rocketDeactivate.svg"
```

Comment:
```suggestion
import { useEffect, useState } from "react";

import getCapsules from "./helpers/api"

import styles from "./AllCapsules.module.css"

import rocketActive from "./assets/rocketActive.svg"
import rocketUnknown from "./assets/rocketUnknown.svg"
import rocketDeactivate from "./assets/rocketDeactivate.svg"
```
Un par de enters pensando en facilitar la lectura